### DISCUSSION TITLE
Getting error when i run the serverpod

### DISCUSSION THREAD

**Discussion by rgustprabha** [serverpod/serverpod#1364] [Q&A]
../../../../.pub-cache/hosted/pub.dev/x509-0.2.3/lib/src/util.dart:102:32: Error: Undefined name 'modulus'.
  return RsaPublicKey(modulus: modulus, exponent: exponent);
  
  
  Docker running in the background and serverpod generate is working fine when i run dart bin/main.dart its giving the above error.. please help me i'm in the delivery stage of the product

**Comment by Isakdl** [comment]
Looks like some issue with a dependency try to clear you pub spec cache `dart pub cache clean` and reinstall the dependencies. Did you change any dependency versions recently? If so that might be the root cause and downgrading back to the previous version might help.
